bazel_tools/grpc-bazel-mingw.patch

```diff
@@ -20,3 +20,4 @@ index 0349e31bb3..d2e7408dbf 100644
 +#define _WIN32_WINNT 0x0A00
  
  #include <windows.h>
+
```

Comment:
Bazel now uses an internal implementation instead of shelling out to `patch`. This internal implementation is a bit less lenient than `patch` and complains about this missing newline.